src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -462,13 +462,6 @@ public static DetailAST parseWithComments(FileContents contents)
     }
 
     @Override
-    public void destroy() {
```

Comment:
@rnveach , TreeWalker have nothing to destroy so he should not override this method and do useless actions. Nobody is inherited from TreeWalker in our project so method is useless.